### PR TITLE
Added danish translation

### PR DESCRIPTION
**Pull request by AlbertCoolGuy**

### CODE REVIEW

**Comment by Epicpkmn11** [comment]
Could you do this on Crowdin? I've added the Danish and here's an invite link: https://crwd.in/universal-updater

It just makes it a bit easier to mange when everything translation related goes through Crowdin

**Comment by AlbertCoolGuy** [comment]
Of course!

**Comment by AlbertCoolGuy** [comment]
I have uploaded the translation to Crowdin.

File: romfs/lang/da/app.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
+{
+	"DOWNLOAD_FAILED":					"Nedhentning mislykkedes!",
+	"NOT_IMPLEMENTED":					"Ikke implementeret endnu.",
+	"DONE":								"Færdig!",
+	"CONNECT_WIFI":						"Forbind venligst til WiFi.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:5]:
```suggestion
	"CONNECT_WIFI":						"Forbind venligst til Wi-Fi.",
```

File: romfs/lang/da/app.json [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+	"ENTER_RED_RGB":					"Indtast den Red RGB.",
+	"ENTER_GREEN_RGB":					"Indtast den Green RGB.",
+	"ENTER_BLUE_RGB":					"Indtast den Blue RGB.",
+
+	"INCOMPATIBLE_SCRIPT":				"Du har et inkompatibelt script.",
+
+	"DELETE_PROMPT":					"Er du sikker på at du vil slette denne Mappe?",
+	"CONFIRM_OR_CANCEL":				"Tryk A for at bekræfte, B for at anullere.",
+
+	"GETTING_SCRIPT_LIST":				"Får script liste...",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:36]:
```suggestion
	"GETTING_SCRIPT_LIST":				"Får script liste…",
```

File: romfs/lang/da/app.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+	"FUTURE_SCRIPT":					"Dette script er et fremtidigt script.",
+	"OUTDATED_SCRIPT":					"Dette script er forældet.",
+	"UP-TO-DATE":						"Dette script er opdateret.",
+
+	"FTP_MODE":							"FTP Mode",
+	"FTP_INITIALIZED":					"FTP Opstarted.",
+	"FAILED_GET_IP":					"Kunne ikke få IP.",
+	"FAILED_INITIALIZE_FTP":			"Opstart af FTP mislykkedes.",
+	"B_FTP_EXIT":						"Tryk B for at gå ud af FTP.",
+	"WIFI_NOT_ENABLED":					"WiFi ikke aktiveret.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:53]:
```suggestion
	"WIFI_NOT_ENABLED":					"Wi-Fi ikke aktiveret.",
```

File: romfs/lang/da/app.json [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+	"SCRIPTCREATOR":					"Script Skaber",
+
+	"CREDITS":							"Credits",
+	"DEVELOPED_BY":						"Udviklet af: Universal-Team.",
+	"MAIN_DEV":							"Hovedudvikler: StackZ",
+	"CURRENT_VERSION":					"Nuværende version: ",
+	"MANY_THANKS":						"Mange Tak Til:",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:61]:
```suggestion
	"MANY_THANKS":						"Mange tak til:",
```

File: romfs/lang/da/app.json [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+	"TRANSLATORS":						"Alle vores oversættere",
+	"HELP_TRANSLATE":					"- For at hjælpe med at oversætte Universal-Updater's tekster.",
+	"HELP_OUT":							"- For at hjælpe med Universal-Updater.",
+	"SCRIPTCREATORS":					"Alle Script Skabere",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:65]:
```suggestion
	"SCRIPTCREATORS":					"Alle script-skabere",
```

File: romfs/lang/da/app.json [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+	"CREATING_SCRIPTS":					"- For at lave scripts til Universal-Updater.",
+	"SHOW_QR":							"Klik her for at vise QR Koden.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:67]:
```suggestion
	"SHOW_QR":							"Klik her for at vise QR-koden.",
```

File: romfs/lang/da/app.json [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+	"LINK":								"Join vores Discord: https://discord.gg/KDJCfGF",
+
+	"RELEASE_TAG":						"Udgivelse Tag: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:70]:
```suggestion
	"RELEASE_TAG":						"Udgivelsestag: ",
```

File: romfs/lang/da/app.json [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+	"DESC":								"Beskrivelse: ",
+	"RELEASE_ID":						"Udgivelse ID: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:72]:
```suggestion
	"RELEASE_ID":						"Udgivelses-ID: ",
```

File: romfs/lang/da/app.json [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+	"TITLE_ID":							"Titel ID: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:73]:
```suggestion
	"TITLE_ID":							"Titel-ID: ",
```

File: romfs/lang/da/app.json [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+
+	"OPEN":								"Åbn",
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
+	"BACK":								"Tilbage",
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
+	"CHANGE_SCRIPTPATH":				"Ændr Script sti",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:86]:
```suggestion
	"CHANGE_SCRIPTPATH":				"Ændr scriptsti",
```

File: romfs/lang/da/app.json [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+	"GET_SCRIPTS_FIRST":				"Få nogle Scripts først!",
+
+	"OPEN":								"Åbn",
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
+	"BACK":								"Tilbage",
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:85]:
```suggestion
	"CHANGE_BAR_STYLE":					"Ændr bar-stil",
```

File: romfs/lang/da/app.json [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+	"DESC":								"Beskrivelse: ",
+	"RELEASE_ID":						"Udgivelse ID: ",
+	"TITLE_ID":							"Titel ID: ",
+	"FILE_SIZE":						"Fil størrelse: ",
+
+	"GET_SCRIPTS_FIRST":				"Få nogle Scripts først!",
+
+	"OPEN":								"Åbn",
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:80]:
```suggestion
	"REFRESH":							"Genindlæs",
```

File: romfs/lang/da/app.json [hunk 13]
```diff
@@ -0,0 +1,173 @@
...
+	"SHOW_QR":							"Klik her for at vise QR Koden.",
+	"LINK":								"Join vores Discord: https://discord.gg/KDJCfGF",
+
+	"RELEASE_TAG":						"Udgivelse Tag: ",
+	"DESC":								"Beskrivelse: ",
+	"RELEASE_ID":						"Udgivelse ID: ",
+	"TITLE_ID":							"Titel ID: ",
+	"FILE_SIZE":						"Fil størrelse: ",
+
+	"GET_SCRIPTS_FIRST":				"Få nogle Scripts først!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:76]:
```suggestion
	"GET_SCRIPTS_FIRST":				"Få nogle scripts først!",
```

File: romfs/lang/da/app.json [hunk 14]
```diff
@@ -0,0 +1,173 @@
...
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
+	"CHANGE_SCRIPTPATH":				"Ændr Script sti",
+	"CHANGE_MUSICFILE":					"Ændr Musik filen",
+	"SELECT_MUSIC_FILE":				"Vælg Musik filen.",
+
+	"VERSION":							"Version: ",
+	"UPDATING":							"Opdaterer...",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:91]:
```suggestion
	"UPDATING":							"Opdaterer…",
```

File: romfs/lang/da/app.json [hunk 15]
```diff
@@ -0,0 +1,173 @@
...
+	"WOULD_YOU_LIKE_UPDATE":			"Vil du gerne opdatere denne Store?",
+	"GET_STORES_FIRST":					"Få nogle Stores først!",
+	"PREPARE_STORE":					"Forbereder store... vent venligst.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:94]:
```suggestion
	"PREPARE_STORE":					"Forbereder store… vent venligst.",
```

File: romfs/lang/da/app.json [hunk 16]
```diff
@@ -0,0 +1,173 @@
...
+	"YES":								"Ja",
+	"NO":								"Nej",
+	"CHANGE_STOREPATH":					"Ændr Store sti",
+	"SELECT_STORE_PATH":				"Vælg Store stien.",
+	"UNKNOWN":							"Ukendt",
+	"ENTER_OWNER_AND_REPO":				"Indtast Ejer og Repo.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:100]:
```suggestion
	"ENTER_OWNER_AND_REPO":				"Indtast "Ejer" og "Repo".",
```

File: romfs/lang/da/app.json [hunk 17]
```diff
@@ -0,0 +1,173 @@
...
+	"ENTER_FULL_URL":					"Indtast det fulde URl.",
+	"SCRIPTS_SUBMENU":					"Scripts Undermenu",
+	"STORE_LIST":						"Store liste",
+	"STORE_SEARCH":						"Store søgning",
+	"UNISTORE_SUBMENU":					"UniStore UnderMenu",
+
+	"OWNER_AND_REPO":					"Ejer & Repo",
+	"FILENAME":							"Filnavn",
+	"OK":								"OK",
+	"DELETE_STORE":						"Vil du gerne slette denne Store?",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:114]:
```suggestion
	"DELETE_STORE":						"Vil du gerne slette denne store?",
```

File: romfs/lang/da/app.json [hunk 18]
```diff
@@ -0,0 +1,173 @@
...
+
+	"DELETE_TITLE":						"Vil du gerne slette denne titel?",
+	"MEDIATYPE_SD":						"MedieType SD",
+	"MEDIATYPE_NAND":					"MedieType NAND",
+	"BOOT_TITLE":						"Vil du gerne starte denne titel?",
+
+	"DISABLE_AUTOBOOT":					"Vil du gerne deaktivere autostart?",
+	"AUTOBOOT_STORE":					"Vil du gerne autostarte denne Store?\n\nDette vil autostarte denne Store ved opstart!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:122]:
```suggestion
	"AUTOBOOT_STORE":					"Vil du gerne autostarte denne Store?\n\nDette vil autostarte denne store ved opstart!",
```

File: romfs/lang/da/app.json [hunk 19]
```diff
@@ -0,0 +1,173 @@
...
+
+	"SCRIPT_NOT_FOUND":					"Dette script eksisterer ikke.",
+	"DELETE_SCRIPT":					"Vil du gerne slette dette script?",
+	"REFRESHING_LIST":					"Genindlæser Liste...",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:126]:
```suggestion
	"REFRESHING_LIST":					"Genindlæser Liste…",
```

File: romfs/lang/da/app.json [hunk 20]
```diff
@@ -0,0 +1,173 @@
...
+
+	"AUTOBOOT_SCRIPT":					"Vil du gerne autostarte dette Script?\n\nDette vil autostarte dette Script ved opstart!",
+	"STARTING_UNIVERSAL_UPDATER":		"Starter Universal-Updater...",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:129]:
```suggestion
	"STARTING_UNIVERSAL_UPDATER":		"Starter Universal-Updater…",
```

File: romfs/lang/da/app.json [hunk 21]
```diff
@@ -0,0 +1,173 @@
...
+
+	"EXECUTE_SCRIPT":					"Vil du gerne udføre dette script",
+	"EXECUTE_STORE":					"Vil du gerne udføre denne entry?",
+
+	"GLOSSARY":							"Ordliste",
+	"DOWNLOAD_ALL":						"Nedhent alle Scripts på en gang.",
+	"CHANGE_VIEW_MODE":					"Ændr ViewModen.",
+	"ENTRY_UP":							"Gå en Entry op.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:137]:
```suggestion
	"ENTRY_UP":							"Gå en entry op.",
```

File: romfs/lang/da/app.json [hunk 22]
```diff
@@ -0,0 +1,173 @@
...
+	"ENTRY_DOWN":						"Gå en Entry ned.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:138]:
```suggestion
	"ENTRY_DOWN":						"Gå en entry ned.",
```

File: romfs/lang/da/app.json [hunk 23]
```diff
@@ -0,0 +1,173 @@
...
+	"REFRESHING_LIST":					"Genindlæser Liste...",
+
+	"AUTOBOOT_SCRIPT":					"Vil du gerne autostarte dette Script?\n\nDette vil autostarte dette Script ved opstart!",
+	"STARTING_UNIVERSAL_UPDATER":		"Starter Universal-Updater...",
+
+	"EXECUTE_SCRIPT":					"Vil du gerne udføre dette script",
+	"EXECUTE_STORE":					"Vil du gerne udføre denne entry?",
+
+	"GLOSSARY":							"Ordliste",
+	"DOWNLOAD_ALL":						"Nedhent alle Scripts på en gang.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:135]:
```suggestion
	"DOWNLOAD_ALL":						"Nedhent alle scripts på en gang.",
```

File: romfs/lang/da/app.json [hunk 24]
```diff
@@ -0,0 +1,173 @@
...
+	"CHANGE_VIEW_MODE":					"Ændr ViewModen.",
+	"ENTRY_UP":							"Gå en Entry op.",
+	"ENTRY_DOWN":						"Gå en Entry ned.",
+	"GO_BACK":							"Gå Tilbage.",
+	"ENTRY":							"Seneste Entry | Sidste Entry",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:140]:
```suggestion
	"ENTRY":							"Seneste entry | Sidste entry",
```

File: romfs/lang/da/app.json [hunk 25]
```diff
@@ -0,0 +1,173 @@
...
+	"REVISION":							"Nutidige Revision | Seneste Revision",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:141]:
```suggestion
	"REVISION":							"Nutidige revision | Seneste revision",
```

File: romfs/lang/da/app.json [hunk 26]
```diff
@@ -0,0 +1,173 @@
...
+	"UPDATE_STORE":						"Opdater den valgte UniStore.",
+	"NOT_FOUND_COLOR":					"Ikke fundet Script Farve",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:143]:
```suggestion
	"NOT_FOUND_COLOR":					"Ikke fundet script-farve",
```

File: romfs/lang/da/app.json [hunk 27]
```diff
@@ -0,0 +1,173 @@
...
+	"OUTDATED_COLOR":					"Forældet Script Farve",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:144]:
```suggestion
	"OUTDATED_COLOR":					"Forældet script-farve",
```

File: romfs/lang/da/app.json [hunk 28]
```diff
@@ -0,0 +1,173 @@
...
+	"UPTODATE_COLOR":					"Opdateret Script Farve",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:145]:
```suggestion
	"UPTODATE_COLOR":					"Opdateret script-farve",
```

File: romfs/lang/da/app.json [hunk 29]
```diff
@@ -0,0 +1,173 @@
...
+	"FUTURE_COLOR":						"Fremtidige Script Farve",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:146]:
```suggestion
	"FUTURE_COLOR":						"Fremtidige script-farve",
```

**Comment by SuperSaiyajinStackZ** [comment]
If you have so many changes, please contribute to crowdin. This isn't a place for suggesting changes.

File: romfs/lang/da/app.json [hunk 30]
```diff
@@ -0,0 +1,173 @@
...
+	"BUTTON_COLOR":						"Knap Farve",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:147]:
```suggestion
	"BUTTON_COLOR":						"Knap-farve",
```
N

File: romfs/lang/da/app.json [hunk 31]
```diff
@@ -0,0 +1,173 @@
...
+	"DELETE_SCRIPT2":					"Slet det valgte Script.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:148]:
```suggestion
	"DELETE_SCRIPT2":					"Slet det valgte script.",
```

File: romfs/lang/da/app.json [hunk 32]
```diff
@@ -0,0 +1,173 @@
...
+	"DELETE_UNISTORE":					"Slet den valgte UniStore.",
+	"REFRESH_SCRIPTBROWSE_PROMPT":		"Vil du genindlæse ScriptBrowseren?",
+	"REFRESH_SCRIPTBROWSE":				"Genindlæs ScriptBrowseren.",
+
+	"DOWNLOAD_ALL_DDM":					"Nedhent Alle",
+	"REFRESH_BROWSE_DDM":				"Genindlæs",
+	"VIEW_DDM":							"Ændr VisningsMåde",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:155]:
```suggestion
	"VIEW_DDM":							"Ændr visningsmåde",
```

File: romfs/lang/da/app.json [hunk 33]
```diff
@@ -0,0 +1,173 @@
...
+	"DELETE_DDM":						"Slet",
+	"UPDATE_DDM":						"Opdater",
+
+	"TAG_NAME":							"Tag navn: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:159]:
```suggestion
	"TAG_NAME":							"Tag-navn: ",
```

File: romfs/lang/da/app.json [hunk 34]
```diff
@@ -0,0 +1,173 @@
...
+	"TARGET":							"Mål: ",
+	"RELEASE_NAME":						"Udgivnings Navn: ",
+	"CREATED_AT":						"Lavet på: ",
+	"PUBLISHED_AT":						"Udgivet på: ",
+	"FETCHING_RELEASES":				"Henter Udgivningerne... vent venligst.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:164]:
```suggestion
	"FETCHING_RELEASES":				"Henter udgivningerne… vent venligst.",
```

File: romfs/lang/da/app.json [hunk 35]
```diff
@@ -0,0 +1,173 @@
...
+	"VERSION_SELECT":					"Vælg Udgivningen du vil nedhente.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:165]:
```suggestion
	"VERSION_SELECT":					"Vælg udgivningen du vil nedhente.",
```

File: romfs/lang/da/app.json [hunk 36]
```diff
@@ -0,0 +1,173 @@
...
+	"IS_PRERELEASE":					"ForUdgivning: ",
+	"DOWNLOAD_ERROR":					"Nedhentings Fejl!",
+	"SCRIPT_CANCELED":					"Script Annuleret!",
+	"SYNTAX_ERROR":						"Syntaks Fejl!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:169]:
```suggestion
	"SYNTAX_ERROR":						"Syntaksfejl!",
```

File: romfs/lang/da/app.json [hunk 37]
```diff
@@ -0,0 +1,173 @@
...
+	"COPY_ERROR":						"Kopierings Fejl!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:170]:
```suggestion
	"COPY_ERROR":						"Kopieringsfejl!",
```

File: romfs/lang/da/app.json [hunk 38]
```diff
@@ -0,0 +1,173 @@
...
+
+	"TAG_NAME":							"Tag navn: ",
+	"TARGET":							"Mål: ",
+	"RELEASE_NAME":						"Udgivnings Navn: ",
+	"CREATED_AT":						"Lavet på: ",
+	"PUBLISHED_AT":						"Udgivet på: ",
+	"FETCHING_RELEASES":				"Henter Udgivningerne... vent venligst.",
+	"VERSION_SELECT":					"Vælg Udgivningen du vil nedhente.",
+	"IS_PRERELEASE":					"ForUdgivning: ",
+	"DOWNLOAD_ERROR":					"Nedhentings Fejl!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:167]:
```suggestion
	"DOWNLOAD_ERROR":					"Nedhentingsfejl!",
```

File: romfs/lang/da/app.json (hunk 36 above)
Comment (comradekingu) [inline romfs/lang/da/app.json:169]:
```suggestion
	"SYNTAX_ERROR":						"Syntaksfejl!",
```

File: romfs/lang/da/app.json [hunk 39]
```diff
@@ -0,0 +1,173 @@
...
+	"SCRIPT_CANCELED":					"Script Annuleret!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:168]:
```suggestion
	"SCRIPT_CANCELED":					"Script annuleret!",
```

File: romfs/lang/da/app.json [hunk 40]
```diff
@@ -0,0 +1,173 @@
...
+	"SYNTAX_ERROR":						"Syntaks Fejl!",
+	"COPY_ERROR":						"Kopierings Fejl!",
+	"MOVE_ERROR":						"Flytnings Fejl!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:171]:
```suggestion
	"MOVE_ERROR":						"Flytningsfejl!",
```

File: romfs/lang/da/app.json [hunk 41]
```diff
@@ -0,0 +1,173 @@
...
+	"DELETE_ERROR":						"Sletnings Fejl!"
```
Comment (comradekingu) [inline romfs/lang/da/app.json:172]:
```suggestion
	"DELETE_ERROR":						"Sletningsfejl!"
```

File: romfs/lang/da/app.json [hunk 42]
```diff
@@ -0,0 +1,173 @@
...
+	"UPDATE_DDM":						"Opdater",
+
+	"TAG_NAME":							"Tag navn: ",
+	"TARGET":							"Mål: ",
+	"RELEASE_NAME":						"Udgivnings Navn: ",
+	"CREATED_AT":						"Lavet på: ",
+	"PUBLISHED_AT":						"Udgivet på: ",
+	"FETCHING_RELEASES":				"Henter Udgivningerne... vent venligst.",
+	"VERSION_SELECT":					"Vælg Udgivningen du vil nedhente.",
+	"IS_PRERELEASE":					"ForUdgivning: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:166]:
```suggestion
	"IS_PRERELEASE":					"Forudgivning: ",
```

File: romfs/lang/da/app.json [hunk 43]
```diff
@@ -0,0 +1,173 @@
...
+
+	"DOWNLOAD_ALL_DDM":					"Nedhent Alle",
+	"REFRESH_BROWSE_DDM":				"Genindlæs",
+	"VIEW_DDM":							"Ændr VisningsMåde",
+	"DELETE_DDM":						"Slet",
+	"UPDATE_DDM":						"Opdater",
+
+	"TAG_NAME":							"Tag navn: ",
+	"TARGET":							"Mål: ",
+	"RELEASE_NAME":						"Udgivnings Navn: ",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:161]:
```suggestion
	"RELEASE_NAME":						"Udgivningsnavn: ",
```

File: romfs/lang/da/app.json [hunk 44]
```diff
@@ -0,0 +1,173 @@
...
+	"SELECT_MUSIC_FILE":				"Vælg Musik filen.",
+
+	"VERSION":							"Version: ",
+	"UPDATING":							"Opdaterer...",
+	"WOULD_YOU_LIKE_UPDATE":			"Vil du gerne opdatere denne Store?",
+	"GET_STORES_FIRST":					"Få nogle Stores først!",
+	"PREPARE_STORE":					"Forbereder store... vent venligst.",
+	"YES":								"Ja",
+	"NO":								"Nej",
+	"CHANGE_STOREPATH":					"Ændr Store sti",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:97]:
```suggestion
	"CHANGE_STOREPATH":					"Ændr store-sti",
```

File: romfs/lang/da/app.json [hunk 45]
```diff
@@ -0,0 +1,173 @@
...
+	"SELECT_STORE_PATH":				"Vælg Store stien.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:98]:
```suggestion
	"SELECT_STORE_PATH":				"Vælg store-stien.",
```

File: romfs/lang/da/app.json [hunk 46]
```diff
@@ -0,0 +1,173 @@
...
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
+	"CHANGE_SCRIPTPATH":				"Ændr Script sti",
+	"CHANGE_MUSICFILE":					"Ændr Musik filen",
+	"SELECT_MUSIC_FILE":				"Vælg Musik filen.",
+
+	"VERSION":							"Version: ",
+	"UPDATING":							"Opdaterer...",
+	"WOULD_YOU_LIKE_UPDATE":			"Vil du gerne opdatere denne Store?",
+	"GET_STORES_FIRST":					"Få nogle Stores først!",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:93]:
```suggestion
	"GET_STORES_FIRST":					"Få nogle stores først!",
```

File: romfs/lang/da/app.json [hunk 47]
```diff
@@ -0,0 +1,173 @@
...
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
+	"BACK":								"Tilbage",
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
+	"CHANGE_SCRIPTPATH":				"Ændr Script sti",
+	"CHANGE_MUSICFILE":					"Ændr Musik filen",
+	"SELECT_MUSIC_FILE":				"Vælg Musik filen.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:88]:
```suggestion
	"SELECT_MUSIC_FILE":				"Vælg musik-filen.",
```

File: romfs/lang/da/app.json [hunk 48]
```diff
@@ -0,0 +1,173 @@
...
+	"OPEN":								"Åbn",
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
+	"BACK":								"Tilbage",
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
+
+	"CHANGE_BAR_STYLE":					"Ændr bar stil",
+	"CHANGE_SCRIPTPATH":				"Ændr Script sti",
+	"CHANGE_MUSICFILE":					"Ændr Musik filen",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:87]:
```suggestion
	"CHANGE_MUSICFILE":					"Ændr musik-filen",
```

File: romfs/lang/da/app.json [hunk 49]
```diff
@@ -0,0 +1,173 @@
...
+	"FILE_SIZE":						"Fil størrelse: ",
+
+	"GET_SCRIPTS_FIRST":				"Få nogle Scripts først!",
+
+	"OPEN":								"Åbn",
+	"SELECT":							"Vælg",
+	"REFRESH":							"Genindlæse",
+	"BACK":								"Tilbage",
+	"FILEBROWSE_MSG":					"Tryk X for at vælge, Vælg for at genindlæse.",
+	"SELECT_SCRIPT_PATH":				"Vælg Script Stien.",
```
Comment (comradekingu) [inline romfs/lang/da/app.json:83]:
```suggestion
	"SELECT_SCRIPT_PATH":				"Vælg scriptstien.",
```

**Comment by comradekingu** [comment]
@Epicpkmn11 @AlbertCoolGuy There are a lot of errors that carry over here. I'd love to help out, but Crowdin is a bridge too far in terms of spying on users. Put the project on Weblate, hosted or otherwise and I'll get on it. (My native language is nb_NO, so some of the changes here are a bit iffy, and not spellchecked).

**Pull request closed without merging**